build.fsx

```diff
@@ -303,6 +303,11 @@ Target "TravisCI" (fun _ ->
   ]
   |> MSBuildRelease "" "Rebuild"
   |> ignore
+  
+  let additionalFiles = 
```

Comment:
Yes sorry, it should be obvious. I can help running / debugging that stuff on mac at some point.

Thanks for giving a hand!